content/en/deployment/strategies/blue-green.md

```diff
@@ -3,78 +3,87 @@ title: "Blue/Green Deployment Strategy"
-
-To complete this quick start, you need the following:
-
-- Access to a Kubernetes cluster where you can install the Remote Network Agent (RNA). This cluster acts as the deployment target for the sample app. You can reuse the cluster from the previous quick starts or create a new one.
-- You need to deploy a [Kubernetes Service object](https://kubernetes.io/docs/concepts/services-networking/service/) that sends traffic to the current version of your application. This is the `trafficManagement.kubernetes.activeService` field in the YAML configuration.
-- (Optional) You can also create a `previewService` Kubernetes Service object so you can programmatically or manually observe the new version of your software before exposing it to traffic via the `activeService`. This is the `trafficManagement.kubernetes.previewService` field in the YAML configuration.
-
-## Add blue/green to your deployment
-
-1. In your deploy file, go to the `strategies` section.
-1. Create a new strategy named `blue-green-deploy-strat` like the following:
-
-   ```yaml
-   strategies:
-    blue-green-deploy-strat:
-      blueGreen:
-        redirectTrafficAfter:
-          - pause:
-              untilApproved: true
-        shutDownOldVersionAfter:
-          - pause:
-              untilApproved: true
-   ```
-
-   See the [Deployment File Reference]({{< ref "reference/deployment/config-file/strategies#bluegreen-fields" >}}) for an explanation of these fields.
-
-   This strategy is configured to pause for manual judgment before redirecting traffic to your new app version as well as before shutting down your old version. You could instead choose to pause for a duration of time.
-
-1. Change the value of `targets.<targetName>.strategy` for one or more of your deployment targets to `blue-green-deploy-strat`.
-
-   ```yaml
-   ...
-   targets:
-    <targetName>:
-      account: <agentIdentifier>
-      namespace: <namespace>
-      strategy: blue-green-deploy-strat
-    ...
-    ```
-1. At the bottom of your file, create a top-level traffic management configuration for your `activeService` and `previewService`:
-
-   ```yaml
-   ...
-   trafficManagement:
-     - targets: ['<targetName>']
-       kubernetes:
-         - activeService: myAppActiveService
-           previewService: myAppPreviewService
-   ...
-   ```
+For this guide, you need the following:
+
+- The [Armory CLI]({{< ref "cli.md" >}}) installed
+- A [Remote Network Agent]({{< ref "remote-network-agent/overview.md" >}}) (RNA) installed in the cluster where you want to deploy your app
+
+
+## Deploy your active service 
+You need to deploy a [Kubernetes Service object](https://kubernetes.io/docs/concepts/services-networking/service/) that sends traffic to the current version of your application. This is the `trafficManagement.kubernetes.activeService` field in the YAML configuration.
```

Comment:
You don't "need" to deploy an active service. It can either exist in cluster, if it does not, you can deploy it along with CD-as-a-Service.

Reply:
Changed this a bit, but happy to adjust more as needed!